apps/meeting/serverless/src/handlers.js

```diff
@@ -151,19 +176,30 @@ exports.join = async (event, context, callback) => {
   }
 
   let meetingInfo = await getMeeting(title);
+  let client = getClientForMeeting(meetingInfo);
   if (!meetingInfo) {
-    const request = {
+    let request = {
```

Comment:
Same here.

Reply:
request will be modified

Reply:
You can still change the property of request when using const. Because the reference of request doesn't change.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/const